### DISCUSSION TITLE
HTTP Method

### DISCUSSION THREAD

**Discussion by dtphap** [orchidsoftware/platform#2022] [General]
Hi,
When following your tutorial "Data Management", I check the route list and see this:
  `GET|HEAD|POST|PUT|PATCH|DELETE|OPTIONS   admin/posts/{method?}  App\Orchid\Screens\Post\PostListScreen@handle`
As I understand, the optional parameter 'method' here is the object method defined in the screen like 'createOrUpdate' or 'remove'
My questions are:

- How can we limit the HTTP methods available to a Screen?
- How can specify HTTP method in a Button?

For example I see that the button 'Delete' send a POST request to the back end with a cumbersome payload including all data of the Blog Post, can we change it to DELETE request with payload just including Blog Post id, and how?

**Comment by AlejandroAkbal** [comment]
Same problem as you, dont know how to do it.
